### DISCUSSION TITLE
Installing and integrating existing project to nhost cli?

### DISCUSSION THREAD

**Discussion by Salman9000** [nhost/nhost#91] [Q&A] [answered]
As the title suggest, I already have a bare flow of a project in in expo react native. How do i integrate it with nhost cli? 
I followed this guide from the documentation:
`If you do not have the above dependencies, download and install the latest release manually from` [GitHub releases](https://github.com/nhost/cli/releases)

After getting the cli.exe, I ran the command from cmd and it asked me to create new app. I don't want to create new app as i already have, i just want to integrate it. How do I go about it?

**Comment by elitan** [comment, marked as answer]
Then you can use `nhost dev` directly to develop locally.

To deploy to production, connect your GitHub repository with your Nhost app and simply and Nhost will automatically deploy your backend for you as soon as you push new changes to GitHub.

**Reply by Salman9000** [reply]
It requires me to login to nhost in the cli. However i logged https://app.nhost.io/login using github. I am unable to login using the github creds in nhost cli

**Reply by elitan** [reply]
## ANSWER

If you've signed in with GitHub you can still set a password for your Nhost account in the Nhost dashboard.

After you've set the password in the Nhost dashboard you use the email and password to sign in with the CLI.

**Reply by Salman9000** [reply]
The change password and reset password feature was not working before. But now it is working. I am able login now. Thanks!

**Reply by elitan** [reply]
That's correct. We shipped it a few days ago. Happy things are working for you now 🙌
